index.html

```diff
@@ -0,0 +1,278 @@
+                    />
+                    <path
+                      id="logo-t"
+                      class="cls-1"
+                      d="M287.85,21.31c1.01,0,1.8,.01,2.38,.04,.58,.02,1,.07,1.26,.14,.26,.07,.43,.16,.5,.25,.07,.1,.11,.19,.11,.29s-.01,.23-.04,.4c-.02,.17-.07,.32-.14,.47-.07,.14-.2,.22-.4,.22-.1,0-.19-.01-.29-.04-.1-.02-.19-.06-.29-.11-.14-.05-.37-.12-.68-.22-.31-.09-.8-.17-1.48-.22-.67-.05-1.63-.07-2.88-.07-.67,0-2.16,.04-4.46,.11s-5.09,.23-8.35,.47c-1.73,4.27-3.47,8.92-5.22,13.93-1.75,5.02-3.25,9.88-4.5,14.58-1.25,4.7-1.87,8.62-1.87,11.74,0,1.2,.13,1.94,.4,2.23,.26,.29,.64,.43,1.12,.43,1.25,0,2.84-.92,4.79-2.77,1.94-1.85,3.92-3.99,5.94-6.41,2.02-2.42,3.79-4.64,5.33-6.66,1.87-2.5,2.93-3.74,3.17-3.74,.05,0,.07,.03,.07,.07,.05,0,.07,.02,.07,.07,0,.33-.77,1.59-2.3,3.78-1.54,2.18-3.42,4.62-5.65,7.31-2.23,2.69-4.49,5.05-6.77,7.09-2.28,2.04-4.17,3.06-5.65,3.06-1.82,0-2.74-1.8-2.74-5.4,0-3.84,.61-8.17,1.84-13,1.22-4.82,2.65-9.53,4.28-14.11,1.63-4.58,3.14-8.53,4.54-11.84-7.49,.91-12.22,1.63-14.18,2.16l-.65,.14c-.14,0-.26-.05-.36-.14l.07,.07s-.1-.02-.14-.07c-.1-.1-.14-.22-.14-.36,0-.48,.67-.91,2.02-1.3,1.34-.38,2.95-.7,4.82-.94s3.64-.43,5.29-.58c1.66-.14,2.87-.24,3.64-.29,.05-.05,.12-.07,.22-.07l2.45-5.62c1.25-2.78,2.16-4.66,2.74-5.62s1.1-1.44,1.58-1.44c.67,0,1.01,.26,1.01,.79,0,.1-.43,1.15-1.3,3.17-.86,2.16-2.02,5.02-3.46,8.57,1.34-.1,3.55-.22,6.62-.36,3.07-.14,5.64-.22,7.7-.22Z"
+                    />
+                  </g>
+                </g>
+              </svg>
+            </h1>
+            <label>
+              <input type="text" class="inputSearch" placeholder="  Sök..." />
+            </label>
+          </div>
+          <nav class="navMenu">
+            <ul>
+              <li><a href="#">Färg</a></li>
+              <li><a href="#">Tapeter</a></li>
+              <li><a href="#">Golv</a></li>
+              <li><a href="#">Inredning</a></li>
+              <li><a href="#">Kundklubb</a></li>
+              <li><a href="#">Kontakt</a></li>
+              <li><a href="#">Om oss</a></li>
+            </ul>
+          </nav>
+          <select class="selectMenu">
+            <option>Meny</option>
+            <option>Färg</option>
+            <option>Tapeter</option>
+            <option>Golv</option>
+            <option>Inredning</option>
+            <option>Kundklubb</option>
+            <option>Kontakt</option>
+            <option>Om oss</option>
+          </select>
+        </div>
+      </header>
+      <section class="firstPage">
+        <article class="containerFirstPageImages">
+          <img
+            id="image1"
+            src="assets/png/image1.png"
+            width="800"
+            height="800"
+            alt="Bild på inredning av ett vardagsrum"
+          />
+          <img
+            id="image2"
+            src="assets/png/image2.png"
+            width="600"
+            height="600"
+            alt="Bild på inredning av en hall"
```

Comment:
Orden "Bild på" kan uteslutas, för en skärmläsare läser ofta upp typ "Image of…" och sen alt-texten, så nu skulle det bli "Image of bild på inredning av en hall" :)

Reply:
Fixat! Tack!